### DISCUSSION TITLE
New sample set: Bygdsiljum Church

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#1747] [General]
I've added a new sample set to the list of available organs for GrandOrgue that I provide. Details can be found on http://familjenpalo.se/vpo/bygdsiljum-church/ and a download link is provided on that page and on the general download page http://familjenpalo.se/vpo/download/ where checksums also can be found to verify download integrity.

Note that this is a _small_ organ, 2 manuals and pedal limited to a total of 11 stops, but the recordings and sample quality of it is second to none. I understand that people will have different opinions on the sound of the organ in itself, and some may indeed not like it, but that's another thing entirely - the quality of it cannot be denied...

Recorded and released at 24 bit, 48 kHz, there are two attacks for all pipes and multiple loops and releases, sampled versions of the tremulant for all affected pipes. Though, the acoustics of that church is very dry, so while the pipes could be recorded with great details I suspect most would like to provide additional reverb to really enjoy the sound.

**Comment by kerkovits** [comment]
Thank you, @larspalo! It was big gift for Christmas. I had no intention to play the organ today, but I came across this sample, and said that I should try it out. I played a few Christmas carols on it, and its beautiful sound made my day happier.

**Comment by rousseldenis** [comment]
@larspalo thanks for this.

Concerning reverb, do you plan to add some?

**Reply by larspalo** [reply]
@rousseldenis I have the IR recordings from Bureå church and Kalvträsk church at http://familjenpalo.se/vpo/ir-recordings already available, and in time I could probably record some more from different churches around here.

Recording IR from different venues, churches, rooms etc. is something that doesn't require so much time, equipment and dedication so that's easily something more people involved with GO could contribute with. We could perhaps even set up a common impulse response repo under GrandOrgue here at GitHub that users could add to.

Otherwise, what you hear in my samples is pretty much how the real room sounds like (at the microphone position actually used) as I don't really do any shortening of the release tails (usually I do add a fade out in the sample production process, but not so much that it would alter the original character perceivably).

Recently I have acquired access to a Zoom F6 to record with, so I'm currently experimenting with recording multi-channel samples at 32 bit float, 96kHz. Can't say for sure what or when the next sample set will be done though...

**Reply by Psi58** [reply]
In case it helps there's a treasure trove of IR recordings here (https://www.openair.hosted.york.ac.uk/?page_id=36). Includes everything from railway tunnels to York Minster!

**Reply by larspalo** [reply]
@Psi58 Thanks for the link! They seem to be working fine, really good fun to hear the Bygdsiljum organ in York Minster acoustics! So, that's a good source to use, but still I think it would be great for GO to have it's own bank of impulse response recordings too.

**Reply by larspalo** [reply]
I've added the link that @Psi58 supplied to the [Wiki page for Impulse Responses](https://github.com/GrandOrgue/grandorgue/wiki/Impulse-Responses) so that it will be more easily found in the future.

**Comment by hnb2907** [comment]
Hi, thanks, the Bygdsiljum organ is brilliant. :)

I've already been using some of the IR impulse responses from openair and a few other places.  The best one I've found is position 3C at Central Hall York University.

**Reply by larspalo** [reply]
@hnb2907 Thanks for testing it! What I think the interesting thing with IR reverb is that it doesn't only add the reverb tail reflections but also actually shape the sound character too. Also as the settings can be varied a lot, even from the same IR file, one can get different results. Using the toolbar release tail shortening feature (test setting it at 250 ms or so) together with the IR reverb also changes the effect!

I just tested using the 3C at Central Hall York University and it indeed does give a nice natural room character without changing the colour of the sound too much.
